packages/codemod/src/plugin-deprecate/isBraidImport.test.ts

```diff
@@ -0,0 +1,37 @@
+import { isBraidImport } from './isBraidImport';
+
+describe('isBraidImport', () => {
+  describe('valid cases', () => {
+    [
+      'braid-design-system',
+      'braid-design-system/css',
+      'braid-src/lib',
+      'braid-src/lib/components',
+      'braid-src/lib/css',
+    ].forEach((source) =>
+      test(`${source}`, () => expect(isBraidImport(source)).toBeTruthy()),
```

Comment:
🏅 Do you prefer this form over `test.each`?

Reply:
Honestly I just forget its there 😄 . I'll update.